lapis/src/main/kotlin/org/genspectrum/lapis/model/mutationsOverTime/MutationsOverTimeModel.kt

```diff
@@ -14,65 +16,91 @@ import org.genspectrum.lapis.silo.SiloAction
 import org.genspectrum.lapis.silo.SiloClient
 import org.genspectrum.lapis.silo.SiloFilterExpression
 import org.genspectrum.lapis.silo.SiloQuery
+import org.genspectrum.lapis.silo.WithDataVersion
 import org.springframework.stereotype.Component
 import java.time.LocalDate
 
-data class MutationOverTimeResponse(
-    var rowLabels: List<NucleotideMutation>,
+data class MutationsOverTimeResult(
```

Comment:
```suggestion
@Schema(description = "The result in tabular format.")
data class MutationsOverTimeResult(
```
(I'm quite sure the text can be improved. Do we want an explanation for rowLabels and columnLabels, too?)